subseas_flex_fcst/layout.py

```diff
@@ -278,15 +278,15 @@ def navbar_layout(phys_units):
             html.Div(
                 [
                     help_layout(
-                        "Lead time",
+                        "Target Date",
                         "lead_time",
-                        "Lead time range choices, in weeks from the start date",
+                        "Target date ranges calculated using lead times and model starts.",
                     ),
                 ],
                 style={
                     "color": "white",
                     "position": "relative",
-                    "width": "105px",
+                    "width": "115px",
```

Comment:
Increase width so that target date fits in full. Also the target date needs to contain the year information.